### PR TITLE
Update some documentation files

### PR DESCRIPTION
**Pull request by nudles**
1. for the git workflow
2. team list
3. release steps

### CODE REVIEW

File: docs-site/docs/contribute-code.md [hunk 1]
```diff
@@ -18,120 +18,70 @@ A simple way to enforce the Google coding styles is to use the linting and forma
...
+  "python.formatting.yapfArgs": ["--style", "{based_on_style: google}"]
 }
 ```
 
 Depending on your platform, the user settings file is located here:
+
 1. Windows %APPDATA%\Code\User\settings.json
-2. macOS $HOME/Library/Application Support/Code/User/settings.json
-3. Linux $HOME/.config/Code/User/settings.json
+2. macOS \$HOME/Library/Application Support/Code/User/settings.json
```
Comment (nudles) [inline docs-site/docs/contribute-code.md]: @FeynmanDNA does  the git hook add the `\` automatically?  I removed it manually; but after committing, it came back again.

File: docs-site/docs/contribute-code.md [hunk 2]
```diff
@@ -18,120 +18,70 @@ A simple way to enforce the Google coding styles is to use the linting and forma
...
-   },
-   "cpplint.cpplintPath": "path/to/cpplint",
-
-   "editor.formatOnSave": true,
-   "python.formatting.provider": "yapf",
-   "python.linting.enabled": true,
-   "python.linting.lintOnSave": true,
-   "clang-format.language.cpp.style": "google",
-   "python.formatting.yapfArgs" : ["--style" , "{based_on_style: google}" ]
+  "[cpp]": {
```
Comment (nudles) [inline docs-site/docs/contribute-code.md]: how many spaces should be added for indent in Docusaurus? 3 or 4?

**Comment by nudles** [comment]
@FeynmanDNA  do we need to fix the node, yarn and Docusaurus  version?
```yarn start:website``` didn't work for me.
```yarn start``` worked.

File: docs-site/docs/contribute-code.md (hunk 1 above)
Comment (FeynmanDNA) [reply docs-site/docs/contribute-code.md]: probably this is due to the linting and formatting, maybe you can try wrap these paths in ticks

2. macOS `$HOME/Library/Application Support/Code/User/settings.json`
3. Linux `$HOME/.config/Code/User/settings.json`

File: docs-site/docs/contribute-code.md (hunk 2 above)
Comment (FeynmanDNA) [reply docs-site/docs/contribute-code.md]: usually, JS files are indenting with 2 spaces
https://github.com/apache/singa-doc/blob/0f8de13fe9decc307fbb8205397edc7650e26017/docs-site/.prettierrc#L7

**Comment by FeynmanDNA** [comment]
> @FeynmanDNA do we need to fix the node, yarn and Docusaurus version?
> `yarn start:website` didn't work for me.
> `yarn start` worked.

if you are at this repo's root, `yarn run start:website` is actually running this
https://github.com/apache/singa-doc/blob/0f8de13fe9decc307fbb8205397edc7650e26017/docs-site/package.json#L54

`yarn start` might work if you are inside the website subdir `docs-site/website`

**Pull request merged**